src/config/pagemanager.php

```diff
@@ -2,8 +2,19 @@
 
 return [
     // Change this class if you wish to extend PageCrudController
-    'admin_controller_class' => 'Backpack\PageManager\app\Http\Controllers\Admin\PageCrudController',
+    'admin_controller_class'        => 'Backpack\PageManager\app\Http\Controllers\Admin\PageCrudController',
 
     // Change this class if you wish to extend the Page model
-    'page_model_class'       => 'Backpack\PageManager\app\Models\Page',
+    'page_model_class'              => 'Backpack\PageManager\app\Models\Page',
```

Comment:
If we are going to have this, will it work for the other (non-unique) page class thingos?

Reply:
I improved the comment. You need to set both config values `'page_model_class'` and `'unique_page_model_class'` if you want to use the same (overridden) model.